crawl-ref/source/dgn-overview.cc

```diff
@@ -607,12 +608,43 @@ bool unnotice_feature(const level_pos &pos)
         || _unnotice_stair(pos);
 }
 
+class dgn_overview : public formatted_scroller
+{
+public:
+    dgn_overview(const string& text = "") : formatted_scroller(FS_PREWRAPPED_TEXT, text) {};
+
+private:
+    bool process_key(int ch) override
+    {
+        switch (ch)
+        {
+            case 'G': do_interlevel_travel(); return false;
```

Comment:
I only realized this now, but `do_interlevel_travel()` calls should be moved out as well, for the same reason; `ctrl-o`, `G`, `?` will currently show the overview screen underneath the travel help popup.

Reply:
You are right, and it looks like `^O`, `_`, `?` behaves the same, too. (and therefore should be moved out as well).